logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -119,7 +121,16 @@ public void setTimestamp(Timestamp t) {
         this.data.put(TIMESTAMP, this.timestamp);
     }
 
+    public Object getFieldForRuby(String reference) {
+        return _getField(reference);
+    }
+
     public Object getField(String reference) {
+        Object val = _getField(reference);
+        return Valuefier.unconvert(val);
+    }
+
+    private Object _getField(String reference) {
```

Comment:
How about I make it public and call it `getUnconvertedField` and call this method from the JRubyExt - because the JRubyExt needs to extract the Ruby Value, which after all is the whole point behind this enhancement. Then I remove `getFieldForRuby`.